ci/pre-2node.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: "Do pre-work to get kubeconfig"
+  hosts: controller
+  vars:
+    ci_framework_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir }}"
+  environment:
+    PATH: "~/.crc/bin:~/.crc/bin/oc:~/bin:{{ ansible_env.PATH }}"
+  tasks:
+      # this task is temperory, eventually, this whole play will be in a separate file, which will do the common tasks already.
+    - name: "Set the sto_dir if it isn't already set"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.projects["github.com/infrawatch/service-telemetry-operator"].src_dir }}'
+      when: sto_dir | default('') | length == 0
+
+    - name: "Run bootstrap playbook"
+      ansible.builtin.shell:
+        cmd: |
+          ansible-playbook -e@{{ ansible_user_dir }}/ci-framework-data/artifacts/parameters/zuul-params.yml {{ ci_framework_dir }}/playbooks/01-bootstrap.yml
+        chdir: "{{ ci_framework_dir }}"
+
+    - name: Run ci_framework infra playbook
+      ansible.builtin.shell:
+        cmd: |
+          ansible-playbook -e cifmw_use_opn=false -e cifmw_use_devscripts=false -e cifmw_basedir={{ ansible_user_dir }}/ci-framework-data/ -e cifmw_openshift_setup_skip_internal_registry_tls_verify=true playbooks/02-infra.yml
+        chdir: "{{ ci_framework_dir }}"
+
+    - name: Run make targets for setup
+      community.general.make:
+        chdir: '{{ ansible_env.HOME }}/{{ zuul.projects["github.com/openstack-k8s-operators/ci-framework"].src_dir }}'
+        target: "{{ item }}"
+      with_items:
+        - setup_tests
+        - setup_molecule
+
+    - name: Check for the kubeconfig file
+      ansible.builtin.shell:
+        cmd: |
+          ls {{ cifmw_openshift_kubeconfig }}
```

Comment:
This is a task that gets read by the user debugging. It can technically be removed. If there is no kubeconfig file at this expected location, the later linking task will fail

Reply:
Actually, this task should fail too, because you get a non-zero return code querying a non-existant file

```
$ ls kubeconfig
kubeconfig
$ echo $?
0
$ ls kubeconfig2
ls: cannot access 'kubeconfig2': No such file or directory
$ echo $?
2

```